struqture/src/mixed_systems/mixed_open_system.rs

```diff
@@ -114,6 +114,177 @@ impl<'a> OpenSystem<'a> for MixedLindbladOpenSystem {
                 actual_number_fermion_subsystems: noise.number_fermions.len(),
             });
         }
+
+        let mut current_number_spins = system.number_spins.clone();
+        let noise_number_spins = noise.number_spins.clone();
+        let noise_number_current_spins = noise.number_spins();
+        let system_number_current_spins = system.number_spins();
```

Comment:
The naming here doesn't seem to be consistent. Also, for line 118 and line 120, based on the name, should the `current_number_spins` method be called?

Reply:
Fixed